docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -101,11 +101,11 @@ Drop the ``--user`` option in the following commands if you use a virtual enviro
 
       pip install --user --upgrade pip
 
-#. Install Ansible dependencies.
+#. Clone required parts of Ansible core for the docs build.
```

Comment:
Should this section be relocated to reflect that it's not a one-time setup, but something that you need to do periodically to get the latest ansible-core source?

Reply:
Thanks for pointing that out. I added it under the building locally section. That duplicates the content but should make it super clear.